### PR TITLE
Changes in stacking.md

### PR DESCRIPTION
**Pull request by Anitaachu**
#### Did you test the changes locally ?
- [ ] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/validators/staking/staking-smart-contract.md [hunk 1]
```diff
@@ -36,9 +36,9 @@ Because this transaction is a call to the Staking smart contract, it passes info
 - `stake` is the name of the smart contract function to be called;
 - `0002` is the number of nodes (unsigned integer, hex-encoded);
 - 45e7131ba....294812f004 is the BLS key of the first node, represented as a 192-character-long hexadecimal string;
```
Comment (camilbancioiu) [inline docs/validators/staking/staking-smart-contract.md:38]: When I wrote this line, I forgot to add backticks here for formatting, please apply this change:
```suggestion
- `45e7131ba....294812f004` is the BLS key of the first node, represented as a 192-character-long hexadecimal string;
```

File: docs/validators/staking/staking-smart-contract.md [hunk 2]
```diff
@@ -36,9 +36,9 @@ Because this transaction is a call to the Staking smart contract, it passes info
...
-- `67656e65736973` is a reserved placeholder, required after each BLS key;
+- `67656e65736973` is the result of signing the address of the sender of the transaction, using the BLS private key of the validator being added to the transaction;
```
Comment (camilbancioiu) [inline docs/validators/staking/staking-smart-contract.md:39]: Consider replacing "validator" with "first node", to maintain consistency with the line above.

Comment (camilbancioiu) [reply docs/validators/staking/staking-smart-contract.md:39]: Also, please replace `67656e65736973` with placeholders like `<BLS-signature-1>` and `<BLS-signature-2>`, to emphasize that these values are different for each node being added to the transaction (due to different keys).

File: docs/validators/staking/staking-smart-contract.md [hunk 3]
```diff
@@ -36,9 +36,9 @@ Because this transaction is a call to the Staking smart contract, it passes info
...
 - `ecf6fdbf5....70f1d251f7` is the BLS key of the second node, represented as a 192-character-long hexadecimal string;
-- `67656e65736973` is the aforementioned reserved placeholder, repeated;
+- `67656e65736973` is the aforementioned the result of signing the address of the sender of the transaction, repeated;
```
Comment (camilbancioiu) [inline docs/validators/staking/staking-smart-contract.md:41]: You can use the same "formula" from above, just repeated here: "<BLS-signature-2> is the result of ... BLS private key of the second node ...".

File: docs/validators/staking/staking-smart-contract.md [hunk 4]
```diff
@@ -93,7 +93,7 @@ Note that:
 
 - `45e7131ba....294812f004` is the BLS key of the first node, represented as a 192-character-long hexadecimal string;
 - `ecf6fdbf5....70f1d251f7` is the BLS key of the second node, represented as a 192-character-long hexadecimal string;
-- no reserved placeholder is needed, as opposed to the staking transaction (see above)
+- no result of signing the address of the sender of the transaction is needed, as opposed to the staking transaction (see above)
```
Comment (camilbancioiu) [inline docs/validators/staking/staking-smart-contract.md:96]: I think it's safe to shorten this to "BLS signature" everywhere. It should be clear from context what this means.

File: docs/wallet/wallet-extension.md [hunk 5]
```diff
@@ -3,10 +3,9 @@ id: wallet-extension
...
 
-Popularly referred to as the "future of money", Maiar now has a robust web extension. Maiar DeFi Wallet is a powerful Elrond web wallet extension that effectively automates interactions with Elrond Decentralized APPs and interacts with Maiar Dex. The Maiar DeFi Wallet enables users to store Elrond tokens such as EGLD, ESDT or NFTs on the Elrond Network.
-
-The Maiar Defi Wallet can be installed on Chrome, Brave, and other chromium-based browsers. The extension is free and secure, with compelling features that allow users to create a new wallet or import existing wallets, manage multiple wallets on the Elrond Mainnet with easy accessibility, and more.
+The Maiar DeFi Wallet can be installed on Chrome, Brave, and other chromium-based browsers. This extension is free and secure, with compelling features that allow you to create a new wallet or import existing wallets, manage multiple wallets on the Elrond mainnet, and store Elrond tokens such as EGLD, ESDT, or NFTs on the Elrond Network with easy accessibility.
```
Comment (camilbancioiu) [inline docs/wallet/wallet-extension.md:8]: :+1:

**Review by camilbancioiu — CHANGES_REQUESTED** [review]
Only minor changes needed.

**Review by camilbancioiu — APPROVED** [review]

File: docs/wallet/wallet-extension.md [hunk 6]
```diff
@@ -3,83 +3,162 @@ id: wallet-extension
...
+
 
 ### Set up Maiar DeFi wallet
 * Once it has been successfully installed, click on the extension to get started.
-* You will be served two options: you can either *Create new wallet* or *Import existing wallet.*
 
-![maiar-defi-example-welcomepage](/wallet/wallet-extension/wallet_extension_step4.png)
+* You will be presented with two options: you can either *Create new wallet* or *Import existing wallet.*
+
+  ![maiar-defi-wallet-welcome-page](/docs.elrond.com/website/static/wallet/wallet-extension/wallet_extension_step4.png)
```
Comment (bogdan-rosianu) [inline docs/wallet/wallet-extension.md:35]: `/wallet/wallet-extension/wallet_extension_step4.png` is enough

File: docs/wallet/wallet-extension.md [hunk 7]
```diff
@@ -3,83 +3,162 @@ id: wallet-extension
...
 
 ## Prerequisites
 
 ### Add Maiar DeFi Wallet to your browser
 
-* On the [Chrome Web Store Extension](https://chrome.google.com/webstore/category/extensions), search for **Maiar DeFi Wallet.** and add it to your browser,
-  ![maiar-defi-chrome](/wallet/wallet-extension/wallet_extension_step1.png)
+* On the [Chrome Web Store Extension](https://chrome.google.com/webstore/category/extensions), search for **Maiar DeFi Wallet.** and add it to your browser. 
 
+  ![maiar-defi-chrome](/docs.elrond.com/website/static/wallet/wallet-extension/wallet_extension_step1.png)
```
Comment (bogdan-rosianu) [inline docs/wallet/wallet-extension.md:18]: `/wallet/wallet-extension/wallet_extension_step1.png`

File: docs/wallet/wallet-extension.md [hunk 8]
```diff
@@ -3,83 +3,162 @@ id: wallet-extension
...
+  
 * Confirm the action in the pop-up.
-  ![maiar-defi-install](/wallet/wallet-extension/wallet_extension_step2.png)
 
+  ![wallet_extension_step2](/docs.elrond.com/website/static/wallet/wallet-extension/wallet_extension_step2.png)
```
Comment (bogdan-rosianu) [inline docs/wallet/wallet-extension.md:22]: `/wallet/wallet-extension/wallet_extension_step2.png`

File: docs/wallet/wallet-extension.md [hunk 9]
```diff
@@ -3,83 +3,162 @@ id: wallet-extension
...
+    
 * You should receive a notification that the extension has been added successfully. 
-  ![maiar-defi-install-pop-up](/wallet/wallet-extension/wallet_extension_step3.png)
+
+  ![wallet_extension_step3](/docs.elrond.com/website/static/wallet/wallet-extension/wallet_extension_step3.png)
```
Comment (bogdan-rosianu) [inline docs/wallet/wallet-extension.md:26]: `/wallet/wallet-extension/wallet_extension_step3.png`

File: docs/wallet/wallet-extension.md [hunk 10]
```diff
@@ -3,83 +3,162 @@ id: wallet-extension
...
-*More information about these networks is contained subsequently in this documentation.*
+
+## Connecting the Maiar DeFi Wallet to Maiar Exchange App 
+
+You can now connect Maiar Exchange to the Maiar DeFi wallet in real-time. With this connection, you will be able to log in to the Maiar exchange using the Maiar DeFi wallet extension in a few steps.
+Follow these steps to proceed:
+
+**Step 1:** To get started, go to the [Maiar Exchange](https://maiar.exchange/) page on the right section of the page, click on the  _“connect”_ button.
+
+![wallet_extension_step17](/wallet/wallet-extension/wallet_extension_step17.PNG)
```
Comment (bogdan-rosianu) [inline docs/wallet/wallet-extension.md:150]: can you please rename the images you have uploaded, for consistency? `PNG` -> `png`

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]
Thanks a lot for this PR!  Small adjustments needed! Also, please retarget this PR towards `development` branch

**Review by bogdan-rosianu — APPROVED** [review]

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]
please solve the conflicts

**Comment by Anitaachu** [comment]
Hi @bogdan-rosianu conflict merged.

**Comment by camilbancioiu** [comment]
I've redirected the PR towards `development`.

**Review by camilbancioiu — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]
